
From: Christoph Hellwig <hch@lst.de>

They've been marked deprecated since 2.5.x and there's no more users.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/module.h |   22 ----------------------
 1 files changed, 22 deletions(-)

diff -puN include/linux/module.h~remove-mod_inc_use_count-mod_dec_use_count include/linux/module.h
--- 25/include/linux/module.h~remove-mod_inc_use_count-mod_dec_use_count	2004-09-26 17:20:53.372301856 -0700
+++ 25-akpm/include/linux/module.h	2004-09-26 17:20:53.376301248 -0700
@@ -567,30 +567,8 @@ struct obsolete_modparm {
 #define MODULE_PARM(var,type)						    \
 struct obsolete_modparm __parm_##var __attribute__((section("__obsparm"))) = \
 { __stringify(var), type };
-
-static inline void __deprecated MOD_INC_USE_COUNT(struct module *module)
-{
-	__unsafe(module);
-
-#if defined(CONFIG_MODULE_UNLOAD) && defined(MODULE)
-	local_inc(&module->ref[get_cpu()].count);
-	put_cpu();
-#else
-	(void)try_module_get(module);
-#endif
-}
-
-static inline void __deprecated MOD_DEC_USE_COUNT(struct module *module)
-{
-	module_put(module);
-}
-
-#define MOD_INC_USE_COUNT	MOD_INC_USE_COUNT(THIS_MODULE)
-#define MOD_DEC_USE_COUNT	MOD_DEC_USE_COUNT(THIS_MODULE)
 #else
 #define MODULE_PARM(var,type)
-#define MOD_INC_USE_COUNT	do { } while (0)
-#define MOD_DEC_USE_COUNT	do { } while (0)
 #endif
 
 #define __MODULE_STRING(x) __stringify(x)
_
